charts/ocis/templates/storageusers/jobs.yaml

```diff
@@ -57,6 +57,22 @@ spec:
                 {{- if  eq .Values.services.storageusers.storageBackend.driver "s3ng" }}
                 - name: STORAGE_USERS_DRIVER
                   value: s3ng
+                - name: STORAGE_USERS_S3NG_METADATA_BACKEND
+                  value: {{ .Values.services.storageusers.storageBackend.driverConfig.s3ng.metadataBackend | quote }}
+                - name: STORAGE_USERS_S3NG_ENDPOINT
+                  value: {{ .Values.services.storageusers.storageBackend.driverConfig.s3ng.endpoint | quote }}
+                - name: STORAGE_USERS_S3NG_REGION
+                  value: {{ .Values.services.storageusers.storageBackend.driverConfig.s3ng.region | quote }}
```

Comment:
Why is the configuration touched in the PR? I don't get the conjunction with the secret.